test/Auth0.OidcClient.Core.UnitTests/Tokens/NoSignatureVerifier.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Auth0.OidcClient.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Auth0.OidcClient.Core.UnitTests.Tokens
+{
+    internal class NoSignatureVerifier : ISignatureVerifier
```

Comment:
unused? other than in tests

Reply:
Yep, literally compiled into a DLL that never gets published to nuget.